src/components/launch/LaunchWindow.module.css

```
@@ -81,3 +81,24 @@
 	background: rgba(255, 255, 255, 0.12);
 	color: #ffffff;
 }
+
+@keyframes recordingPulse {
```

Stylelint (17.7.0)

[error] Expected keyframe name "recordingPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)